src/main/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -63,6 +63,7 @@ class StockMovement implements Validateable{
 
     StockMovementDirection stockMovementDirection
     StockMovementStatusCode stockMovementStatusCode
+    StockMovementType stockMovementType
```

Comment:
Why this was added here if it is not populated (keep in mind this StockMovement is not backed by the stock-movement view)

Reply:
It has been added in case we returned the `StockMovement` to the show.gsp, not the `OutboundStockMovement`.
Since this property was missing in the `StockMovement`, I would get an exception when trying to evaluate this part of the code:
```gsp
 <g:if test="${stockMovement?.stockMovementType == StockMovementType.RETURN_ORDER && isInbound}">
```
because either `StockMovement` or `OutboundStockMovement` might hide under the `stockMovement`.

I know it is not populated and it would return `null` for any case, we don't set it explicitly, but in this case we would want it to just evaluate to false, instead of throwing the exception about the missing method.